playbooks/aws/provisioning_vars.yml.example

```diff
@@ -14,12 +14,8 @@
 # Deployment type must be specified.
 openshift_deployment_type: # 'origin' or 'openshift-enterprise'
 
-# openshift_release must be specified.  Use whatever version of openshift
-# that is supported by openshift-ansible that you wish.
-openshift_release: # v3.7
-
 # This will be dependent on the version provided by the yum repository
-openshift_pkg_version: # -3.7.0
+#openshift_pkg_version: # -3.10.0
```

Comment:
i guess you want `3.11` since is in master branch and `3.10` for `release-3.10` branch ?

Reply:
I dunno.  I just bounced it up to 3.10 since I test using cherry-pick to release-3.10 branch.  I haven't been able to deploy 3.11 yet.